src/pages/Components/Notification/code.md

```diff
@@ -0,0 +1,51 @@
+---
+title: 'Code'
+order: 2
+---
+
+## Import
+
+```css
+@import 'settings-tools/_all-settings';
+@import 'components/_c.notification';
+```
+
+## Basic usage
+
+Wrap a h3, p and a a or button tags in a div and apply the `mc-notification` class:
+
+```html
+  <div class="mc-notification">
+    <h3>Notification title</h3>
+    <p>
+      Lorem ipsum dolor sit amet, consectetuer adipiscing elit. Aenean commodo ligula eget dolor. Aenean massa.
+    </p>
+    <a href="#">Link</a>
+  </div>
+```
+
+<preview path="src/pages/Components/Notification/previews/notification"></preview>
+
+## States
+
+You have 4 states of notification available (self-explanatory): *information*(default), *success*, *warning* and *danger*. Apply them using the corresponding class: `mc-notification-[type]`.
+
+<preview path="src/pages/Components/Notification/previews/notification-states"></preview>
```

Comment:
Please delete the italic information.  It can be confusing for users.
But, We can rename the title "Notification title" by states name :
- "Information notification" instead of "INFORMATION (default) Notification title"
- "Success notification" instead of "SUCCESS Notification title"
- "Warning notification" instead of "WARNING Notification title"
- "Danger notification" instead of "DANGER Notification title"

Reply:
I see an another problem with the icons, there is different thickness between "Information - Danger" and "Succes - Warning".
Do you use the good icons ?
Information : *Navigation_Notification_Information_32px*
Succes : *Navigation_Notification_Avaible_32px*
Warning : *Navigation_Notification_Warning_32px*
Danger : *Navigation_Publish_CircleDelete_32px*